frontend/model/notifications/templates.js

```diff
@@ -12,6 +12,33 @@ const escapeForHtml = (text) => text.replace(/[<>&]/g, '\\$&')
 const strong = (text) => `<strong>${escapeForHtml(text)}</strong>`
 
 export default ({
+  GENERAL (data: { contractID: string, message: string }) {
+    return {
+      body: data.message,
+      icon: 'cog',
+      level: 'info',
+      linkTo: '',
+      scope: 'app'
+    }
+  },
+  WARNING (data: { contractID: string, message: string }) {
+    return {
+      body: data.message,
+      icon: 'exclamation-triangle',
+      level: 'danger',
+      linkTo: '',
+      scope: 'app'
+    }
+  },
+  ERROR (data: { contractID: string, message: string }) {
+    return {
+      body: data.message,
+      icon: 'exclamation-triangle',
+      level: 'danger',
+      linkTo: `/app/dashboard?modal=UserSettingsModal&section=application-logs&errorMsg=${encodeURI(data.message)}`,
+      scope: 'app'
+    }
+  },
```

Comment:
At first guess I would say that errors and warnings would rather belong in the global banner or something similar. But maybe it's already used for too many things.